Extensions/blacklist.js

```diff
@@ -503,18 +503,24 @@ XKit.extensions.blacklist = new Object({
 						tag_array.push($(this).html().replace("#","").toLowerCase());
 					});
 				}
+				if ($(this).find(".mh_post_tags").length > 0) {
+					$(this).find(".mh_post_tags a").each(function() {
+						tag_array.push($(this).html().replace("#","").toLowerCase());
+					});
+				}
+
 
 				//alert(tag_array);
 
 				// Collect the title contents too.
 				var m_title = "";
-				if ($(this).find(".post_title").length > 0) {
-					m_title = $(this).find(".post_title").html();
+				if ($(this).find(".post_title, .post-title").length > 0) {
+					m_title = $(this).find(".post_title, .post-title").html();
 				}
 
 				// Collect the author info, if the option is toggled.
 				var m_author = "";
-				if (XKit.extensions.blacklist.preferences.check_authors.value) {
```

Comment:
i disabled this on mobile because existing code was throwing errors on mobile, i found the bug though, so it'll be enabled on next commit